examples/text_to_image/requirements_sdxl.txt

```diff
@@ -1,4 +1,4 @@
-accelerate>=0.16.0
+accelerate>=0.22.0
```

Comment:
Is this necessary?

Reply:
Unfortunately, without this change, there is still a spike in GPU memory during checkpointing :/. From Zach's message on the issue, my understanding is that there was a bug when saving with the accelerator while using mixed precision